packages/framework/src/Resources/config/services.yml

```diff
@@ -995,8 +995,12 @@ services:
         calls:
             - method: setBuildVersion
               arguments:
-                  - ['%build-version%']
+                  - '%build-version%'
```

Comment:
This looks like a change that should have been in previous commit.

Reply:
I'll squash the two commits